src/app/app-routing.module.ts

```diff
@@ -27,7 +28,9 @@ const routes: Routes = [
   {path: 'hotel', component: HotelComponent},
   {path: 'hotel/edit', component: EditHotelComponent},
   {path: 'filter', component: FilterComponent},
-  {path: 'filter-hotel', component: FilterHotelComponent}
+  {path: 'filter-hotel', component: FilterHotelComponent},
+  {path: '404', component: NotFoundComponent}
```

Comment:
What about any other routes like '/this-route-doesnt-exist' or '/how-you-gonna-handle-me' ?